### DISCUSSION TITLE
airflow2.7.3 hiveoperator is running continuously

### DISCUSSION THREAD

**Discussion by shixiutao** [apache/airflow#35829] [Q&A]
I started Airflow version 2.7.3 using standalone mode, and then I wrote a simple hiveoperator to execute it, which was still running; The connection I am using is hive_ Cli, and I can successfully execute the beeline command printed in the log on the machine where Airfow is installed

![image](https://github.com/apache/airflow/assets/8542648/394d7847-a1af-49cd-8f3b-93cbac5da4f8)

**Comment by potiuk** [comment]
Looks like you are using cloudera account - this looks like an issue that you should raise to their paid support.

**Reply by shixiutao** [reply]
When I use other connections, such as hiveserver2 or metaserver, he reports an error like this
![image](https://github.com/apache/airflow/assets/8542648/19170cf5-1e6d-4f11-a197-55b94f55e127)

**Reply by potiuk** [reply]
If you ask me - no idea. But maybe someone could help and maybe asking your paid support from cloudera would be a good idea.
